script/centos-build

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -eu
+#
+# This script works with CentOS 6 or 7
+# The CentOS 5 kernel is too old for go's liking.
+
+trap 'echo FAIL' ERR
+
+if grep -q ' 6' /etc/redhat-release; then
+  rpm -q epel-release || rpm -Uvh http://download.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
+fi
+
+yum install -y bison git golang make man
+cd /tmp
+[ -d git-lfs ] || git clone https://github.com/jsh/git-lfs
+cd git-lfs
```

Comment:
#What if there line 13-15 you did instead
if git rev-parse; then
cd $(git rev-parse --show-toplevel)
else
cd /tmp
[ -d git-lfs ] || git clone https://github.com/github/git-lfs
cd git-lfs
fi
#That way it will use someone else's fork/branch for testing purposes? with out having to create a symlink to their repo from /tmp